src/App.js

```diff
@@ -1,24 +1,12 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Route } from "react-router-dom";
+import LandingPage from "./components/LandingPage/LandingPage.js"
+
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Route path="/landing" component={LandingPage} />
```

Comment:
I guess it should be the landing... The reason I did "/landing" was just in case our team prefers for "/" path to be the marketing page. I'll talk to them tonight and get it sorted out on my next PR. I wanted to have this merged into master as soon as possible because our other team members need the images in order to finish working on their components.

Reply:
Okay, gotcha - I see there's both a Marketing Page and Landing page in the UX Designer's prototype which is.. interesting 🤔 

We just need to do a quick rebase and we can merge this in 👍